BUILD.osx.md

```diff
@@ -45,7 +56,8 @@ Alternatively if using Qt6 from Homebrew, do
 
     cmake -Bbuild -H. -GNinja -DCMAKE_PREFIX_PATH=/usr/local/opt/qt6
 
-or, if on Apple silicon, brew will store the Qt binaries in a different location. Additionally, OpenSSL will be in a similar location; `/opt/homebrew/Cellar/openssl@3`, which can be set in the project level `CMakeLists.txt` file.
+or, if on Apple silicon, brew will store the Qt binaries in a different location. Additionally, OpenSSL will be in a
```

Comment:
Can remove mentions of installing Qt via brew, since brew does not carry the correct version.